package.json

```diff
@@ -38,7 +38,7 @@
   "dependencies": {
     "@dcl/explorer": "^1.0.73952-20230125122546.commit-2029f5d",
     "@dcl/feature-flags": "^1.1.0",
-    "@dcl/kernel-interface": "^2.0.0-20210922153939.commit-017905d",
+    "@dcl/kernel-interface": "^2.0.0-20230512115658.commit-b582e05",
```

Comment:
Do we manually need to update this every time kernel changes?

Reply:
Yes, this is the kernel-interface, not kernel. It's the connection between the website and kernel (browser-interface)

Here is the repo: https://github.com/decentraland/kernel-interface